<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edicao</class>
 <widget class="QMainWindow" name="Edicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>347</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>347</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edição</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 130, 195);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_edite_dados">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Edite os dados do usuário</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="novo_nome">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>Novo nome</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="novo_usuario">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="placeholderText">
       <string>Novo usuário</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLineEdit" name="nova_senha">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>170</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
background-color: #fff;
border-radius: 15px;
padding: 5px;
padding-left: 7px;
}
QLineEdit:focus{
border: 2px solid #edd400;
}</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Nova senha</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="bl_admin">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
color:#fff;
}
QCheckBox::Checked{
	color: #edd400;
}</string>
      </property>
      <property name="text">
       <string>Administrador</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_erro">
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);;
font-size: 13pt;
font-weight: bold;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_salvar">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
background-color: #6EC8DB;
border-radius: 15px;
color: white;
}
QPushButton:hover{
background-color:#60afbf;
}</string>
         </property>
         <property name="text">
          <string>SALVAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancelar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
background-color: #df4759;
border-radius: 15px;
color: #fff;
}
QPushButton:hover{
background-color: #ba3847;
}

</string>
         </property>
         <property name="text">
          <string>CANCELAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
